### ISSUE TITLE
Does Tomofast support cross-gradient inversion using velocity model？

### ISSUE THREAD

**Issue by WanZiXuan** [TOMOFAST/Tomofast-x#6]
Hi,
   Thanks for sharing the code! I'm tring to add structual similarity constaints into gravity anomaly  inversion using velocity model, however, the program seem to only support cross-gradient inversion between grav and magnetic data.  I'd like to konw if Tomofast support cross-gradient inversion using other type of geophysics model?

**Comment by vogarko** [comment]
Yes, you can use the magnetic problem for velocity model, and set you velocity to magnetic starting model. To achieve this use the following Parfile parameters:

inversion.startingModeling.type=2
inversion.startingModeling.magn.file= Your velocity model
inversion.startingModeling.grav.file=  Model with zeros

Also use the following setting for the cross gradient constraint:
inversion.crossGradient.magn.keepModelConstant = 1

And set the low problem weight for the mag problem (to ignore its data term):
inversion.joint.grav.problemWeight  = 1.d0
inversion.joint.magn.problemWeight  = 1.d-30

**Comment by WanZiXuan** [comment]
Thanks for your reply！I will try it.
